lib/Images.php

```diff
@@ -53,21 +68,39 @@ public static function extractImageUrl($imageField)
         return $image ? $image->url : null;
     }
 
+    public static function getHeroImageCrops($imageUrl)
```

Comment:
This name feels a little different to what it does, not used on a hero image, being used on the trail photo? Maybe a more general name?

Reply:
Good call, it's not super clear. Just renamed it to `getStandardCrops()` to imply they're the image sizes we use generally.